src/App.js

```diff
@@ -1,15 +1,77 @@
-import SearchBar from './components/SearchBar/SearchBar';
-import Users from './components/Users/Users';
-import './App.css';
+import { useEffect, useState } from "react";
+import { Button } from "react-bootstrap";
+import SearchBar from "./components/SearchBar/SearchBar";
+import Users from "./components/Users/Users";
+import axios from "axios";
+import "./App.scss";
 
 function App() {
+  const API = process.env.REACT_APP_API_URL;
+  const [users, setUsers] = useState([]);
+  const [searchInput, setSearchInput] = useState("");
+  const [expanded, setExpanded] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState("");
+
   // TODO: Fetch data here
+  useEffect(() => {
+    getUsers();
```

Comment:
Nice, I like how you defined the function outside of `useEffect()`. That looks very clean.